test/GithubApi.Repositories.test.js

```diff
@@ -0,0 +1,78 @@
+const axios = require('axios');
+const chai = require('chai');
+const md5 = require('md5');
+
+const { expect } = chai;
+const { StatusCodes } = require('http-status-codes');
+chai.use(require('chai-subset'));
+
+const urlBase = 'https://api.github.com/users';
+const owner = 'aperdomob';
+
+describe('Stage 8 - GET: Repositories testing', () => {
+  let userResponse;
+  before(async () => {
+    userResponse = await axios.get(`${urlBase}/${owner}`);
+  });
+
+  it('2. API consume', async () => {
+    expect(userResponse.status).to.equal(StatusCodes.OK);
+    expect(userResponse.data.name).to.equal('Alejandro Perdomo');
+    expect(userResponse.data.company).to.equal('Perficient Latam');
+    expect(userResponse.data.location).to.equal('Colombia');
+  });
+
+  describe('Using hypermedia', () => {
+    let repositoriesResponse;
+    let repository;
+    before(async () => {
+      repositoriesResponse = await axios.get(`${userResponse.data.repos_url}`);
+      repository = repositoriesResponse.data.find(
+        (repo) => repo.name === 'jasmine-json-report'
+      );
+    });
+
+    it('3. Repositories List', async () => {
+      const expectedResponse = {
+        name: 'jasmine-json-report',
+        private: false,
+        description: 'A Simple Jasmine JSON Report'
+      };
+
+      expect(repositoriesResponse.status).to.equal(StatusCodes.OK);
+      expect(repository).to.containSubset(expectedResponse);
+    });
+
+    it('4. Repo download', async () => {
+      const zipFile = await axios.get(`${repository.url}/zipball/${repository.default_branch}`);
+
+      expect(zipFile.status).to.equal(StatusCodes.OK);
+      expect(zipFile.headers['content-type']).to.equal('application/zip');
+    });
+
+    let readmeFile;
+    before(async () => {
+      readmeFile = await axios.get(`${repository.url}/readme`);
+    });
```

Comment:
the idea is to create a new describe here, that will contain this before, and also, the next two its, that are related with the readme file